### DISCUSSION TITLE
A Cloud Development Environment with Quasar

### DISCUSSION THREAD

**Discussion by hyberson** [quasarframework/quasar#15323] [General - Components / Directives / etc] [answered]
Hello.

I would like to create a **cloud** development environment in order to learn how to use Quasar.

I am not familiar with development tools or development environments.

Please point me in the right direction.

I appreciate the help.

**Comment by MilosPaunovic** [comment, marked as answer]
What do you mean by `cloud development environment`? 

If you want to have an environment online for testing Quasar, you can choose one of the existing platforms, from the list below:
![image](https://user-images.githubusercontent.com/62475782/214588606-a8e73846-ab7f-41e1-ae4c-b7495ccd16f0.png)

**Reply by hyberson** [reply]
Hello, Milos.
Thanks for your answer.
I do not want to have VS Code installed on a local machine (PC/laptop). I do not want to have source code stored on a local machine at any time.

**Comment by yusufkandemir** [comment]
https://quasar.dev/start/quasar-cli#trying-quasar-cli-online
> You can try Quasar CLI online directly in the browser, without installing anything! You will also be able to use the command line, so it will be almost identical to creating a project locally.
> 
> [**Open in StackBlitz**](https://stackblitz.quasar.dev/)

**Reply by hyberson** [reply]
Hello, Yusuf.
Thank you for your answer.

**Comment by hyberson** [comment]
This is what I was looking for, when I asked about "cloud development environment": https://www.freecodecamp.org/news/github-codespaces-vs-gitpod-cloud-based-dev-environments/.
